ModuleOneSecondTaskLibrary/GreatingBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="GreatingBuilder.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using System;
+
+namespace ModuleOneSecondTaskLibrary
+{
+    /// <summary>
+    /// Builds greating text.
+    /// </summary>
+    public class GreatingBuilder
+    {
+        /// <summary>
+        /// Returns greating in format:
+        /// «{current_time} Hello, {name}!».
+        /// </summary>
+        /// <returns>Greating text.</returns>
+        public string Build()
+        {
+            var args = Environment.GetCommandLineArgs();
+            var name = args?.Length > 1 ? args[1] : Environment.UserName;
```

Comment:
A great find for the default value (`Environment.UserName`), but why `args[1]` and not `args[0]`?

Reply:
The zero-argument is the name of the program. So, we should start with the first one.